app/dashboard/views.py

```diff
@@ -1006,6 +1006,34 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
     return TemplateResponse(request, 'bounty/details.html', params)
 
 
+def get_notify_funder_modal(request, stdbounties_id):
+    stdbounties_id = clean_str(stdbounties_id)
+    bounty = Bounty.objects.filter(standard_bounties_id=stdbounties_id).first()
```

Comment:
If we use `network` and `standard_bounties_id`, we can use `Bounty.objects.current().filter(network=network, standard_bounties_id=stdbounties_id)`.

`.current()` makes sure that you get the latest revision of that bounty object 👍